modules/reactor/src/meshgenerators/SimpleHexagonGenerator.C

```diff
@@ -125,5 +125,7 @@ SimpleHexagonGenerator::generate()
         _external_boundary_name;
   }
 
+  mesh->prepare_for_use();
```

Comment:
`ReplicatedMesh::stitching_helper()` checks `libmesh_assert_equal_to (_is_prepared, other_mesh._is_prepared);` and throws an error in DBG mode if this mesh is "not prepared". This can be either put here or before stitching in `PatternedHexMG`.